showkase-processor/src/main/java/com/airbnb/android/showkase/processor/ShowkaseProcessor.kt

```diff
@@ -50,18 +52,19 @@ class ShowkaseProcessor @JvmOverloads constructor(
     private val logger = ShowkaseExceptionLogger()
     private val showkaseValidator = ShowkaseValidator()
 
-    override fun getSupportedAnnotationTypes(): MutableSet<String> = mutableSetOf(
-        ShowkaseComposable::class.java.name,
-        PREVIEW_CLASS_NAME,
-        ShowkaseColor::class.java.name,
-        ShowkaseTypography::class.java.name,
-        ShowkaseRoot::class.java.name,
-        ShowkaseScreenshot::class.java.name,
-    )
-
-    override fun getSupportedOptions(): MutableSet<String> {
-        return mutableSetOf("skipPrivatePreviews")
+    override fun getSupportedAnnotationTypes(): MutableSet<String> {
+        val supportedAnnotationTypes = mutableSetOf(
```

Comment:
you can just inline this instead of creating a separate variable

Reply:
Good point! I will fix that! Thanks!

Reply:
Fixed in 855cd00 :)